packages/components/src/Toast/Toast.test.tsx

```diff
@@ -36,49 +35,70 @@ const infoMessage = "Bland Toast";
 const errorMessage = "Errorful should last inbetween min-max";
 
 it("creates the placeholder div on showToast call", () => {
-  const { getByText } = render(<MockToast />);
-  expect(document.querySelector(`#atlantis-toast-element`)).not.toBeInstanceOf(
-    HTMLDivElement,
-  );
```

Comment:
This is testing if the container is not there at first. But now, as this bits were moved out of `showToast`, they're rendering since the beginning. Should we change the test or make it work like it was before?

Reply:
yeah, since it's specifically testing instead of implying